scripts/linux/trcE2ETest.sh

```diff
@@ -757,6 +757,12 @@ function run_connectivity_test() {
             fi
 
             sleep "${sleep_frequency_secs}s"
+
+            #### DO NOT MREGE- Temporarily added for quicker debugging
```

Comment:
Just checking if this needs to be removed?

Reply:
Yes. Sorry, I should have marked this as DRAFT. I'll be sure to revert this once this PR is ready for review.